.github/workflows/accessibility-testing.yml

```
@@ -0,0 +1,241 @@
+          configPath: './.github/lighthouse-config.json'
+          uploadArtifacts: true
+          temporaryPublicStorage: true
+
+  a11y-summary:
+    name: Accessibility Summary
+    needs: [pa11y-test, axe-test, lighthouse-a11y]
+    if: always()
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate Summary
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)